src/bot/conversations/main_application.py

```diff
@@ -4,6 +4,13 @@
 from bot.constants.info.text import START_MESSAGE
 
 
+MAIN_MENU = range(1)
```

Comment:
Давай перенесем это в файл state.py
```suggestion
(
    MAIN_MENU,
    BACK
) = range(2)
```